testing/tests/audio.lua

```diff
@@ -0,0 +1,296 @@
+-- love.audio
+
+
+-- love.audio.getActiveEffects
+love.test.audio.getActiveEffects = function(test)
+  -- tests
+  test:assertNotEquals(nil, love.audio.getActiveEffects(), 'check not nil')
+  test:assertEquals(0, #love.audio.getActiveEffects(), 'check no effects running')
+  love.audio.setEffect('testeffect', {
+    type = 'chorus',
+    volume = 10
+  })
+  test:assertEquals(1, #love.audio.getActiveEffects(), 'check 1 effect running')
+  test:assertEquals('testeffect', love.audio.getActiveEffects()[1], 'check effect details')
+end
+
+
+-- love.audio.getActiveSourceCount
+love.test.audio.getActiveSourceCount = function(test)
+  -- tests
+  test:assertNotEquals(nil, love.audio.getActiveSourceCount(), 'check not nil')
```

Comment:
I think the `assertEquals` below (and in many other tests) already covers situations where the function returns nil.